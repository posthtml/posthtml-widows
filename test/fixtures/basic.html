skips unmarked strings

<p class="one two" prevent-widows>
  <span>lorem ipsum dolor</span>
</p>
