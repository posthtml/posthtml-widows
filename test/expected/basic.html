skips unmarked strings

<p class="one two">
  <span>lorem ipsum&nbsp;dolor</span>
</p>
